node/src/external/wrapper/node_wrapper.h

```diff
@@ -0,0 +1,271 @@
+// See LICENSE file
+
+#ifndef SRC_JS_HEADER_
+#define SRC_JS_HEADER_
```

Comment:
The node-apphost repo should post GitHub releases (which are versioned) containing all the binaries, and probably these headers also. Then a script here should download a specific version. When we want to update to depend on a newer version of headers and binaries, we update the downloaded release version in the script. Think of it as equivalent to an NPM package dependency, except it's not managed via NPM because the binaries are too large.